Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Requirements for container registry
+type: Decision Record
+status: Draft
+track: KaaS
+---
+
+# Introduction  
+
+A container registry is a repository or collection of repositories used to store and 
+access container images. Container registries are used by the teams to store the 
```

Comment:
The second sentence here feels a bit redundant to me, especially since `the teams` have not yet been introduced.

Reply:
Thank you for all your valuable comments! All have been addressed by a fixup commit 079b16291f89aa087840ec8f1b40bee3040c9a06

Reply:
Cheers, thank you for your kind words! I'll mark all conversation as resolved.